rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -1430,4 +1430,11 @@ public void addEmbeddedClearingTeam(HalResource<Project> userHalResource, String
         if(sw360User!=null)
             addEmbeddedUser(userHalResource, sw360User, resource);
     }
+
+    public void addEmbeddedOtherLicenses(HalResource<Release> halComponent, Set<String> licenseIds) {
```

Comment:
change it to halRelease from halComponent.

Reply:
@rudra-superrr fixed it